packages/devtools_app/lib/src/screens/memory/allocation_profile_table_view_controller.dart

```diff
@@ -0,0 +1,119 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
```

Comment:
Can you move the tab 'Profile' related code to panes/profile (the same way as Leaks and Diff).
Thus, after retirement of 'Allocation' and 'Anazysis', each tab will be live in a separate folder, that will make it easier to support the code.